FutureNorthants/northants-website
# Clearing up various warnings

### Testing
Starting the front end and just viewing the homepage will trigger a number of warnings that this PR fixes or shuts up. It's only the home page that actually has a code change so...
1. View the home page and verify unchanged from before.
2. Set to memorial mode and verify home still works (with and without a hero image).
3. To verify that the other files have been shut up, just look at the output log in your frontend server terminal window or browser console (in Firefox at least it was pretty noisy before).
